profile/setup.md

```diff
@@ -109,3 +109,23 @@ npm install
 cd ..
 done
 ```
+
+## Sharing Screen
+Sharing screen is disabled if your build uses wayland. To circumvent this do the following:
+Open terminal
+Check if system uses wayland:
+```
+echo $XDG_SESSION_TYPE
+```
+If the command outputs wayland continue with
+```
+sudo nano /etc/gdm3/custom.conf
+```
+and uncommit this line:
+```
+#WaylandEnable=false
+```
+Save and exit, reboot and checking with first command should display
```

Comment:
Save, exit and then reboot the computer. If you check with first command again, it should display: